lib/cloudant.js

```diff
@@ -140,19 +140,18 @@ Cloudant.prototype.fromDB = function(modelName, modelObject, doc) {
 };
 
 /**
- * Create a new model instance for the given data
+ * Insert a model instance
  *
  * @param {String} model The model name
  * @param {Object} data The model data
  * @param {Function} [cb] The cb function
  */
-Cloudant.prototype.create = function(model, data, options, cb) {
-  debug('Cloudant.prototype.create %j %j %j ', model, data, options);
+Cloudant.prototype._insert = function(model, data, cb) {
   var self = this;
   var idName = self.idName(model);
   var mo = self.selectModel(model);
   mo.db.insert(self.toDB(model, mo, data), function(err, result) {
-    debug('Cloudant.prototype.create insert %j %j', err, result);
+    debug('Cloudant.prototype.insert %j %j', err, result);
```

Comment:
Nitpick:

I would probably make them more explicit like:
```
debug('Cloudant.prototype._insert err:%j, result: %j', err, result);
```

Please feel free to ignore it